dot/rpc/subscription/websocket.go

```diff
@@ -209,6 +212,12 @@ func (c *WSConn) initStorageChangeListener(reqID float64, params interface{}) (u
 	return myObs.id, nil
 }
 
+func (c *WSConn) unsubscribeStorageListener(params interface{}) {
+	id := params.([]interface{})[0].(float64)
```

Comment:
Would be nice to have some type checking here to ensure we don't face nil pointer or interface conversion panic?

Reply:
I agree, should return an error if the type check fails.  At least we can log it out in the unlikely case it happens.